### DISCUSSION TITLE
启动exe就闪一下，然后就没反应了

### DISCUSSION THREAD

**Discussion by mjy66** [mikumifa/biliTickerBuy#129] [提问专区]
而且找不到error.log咋办，也用directx_repair检查了

**Comment by mikumifa** [comment]
window？

**Reply by mjy66** [reply]
是window系统

**Comment by mikumifa** [comment]
会不会python启动

**Reply by mjy66** [reply]
会一点

**Reply by mjy66** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/173073274/d1212d6b-a242-4556-a3f5-524af636209b)
不知道是不是这样启动，好像没报错，感觉也没反应

**Reply by mikumifa** [reply]
好可爱😁😁

**Reply by mikumifa** [reply]
就是你下载我的项目，然后用pycharm运行 main.py

**Comment by mikumifa** [comment]
python试一试

**Comment by xiyuchengxu** [comment]
驱动
